net/seafile-seahub/Makefile

```diff
@@ -24,16 +24,15 @@ include ../../lang/python/python-package.mk
 
 SEAFILE_PYTHON_DEPENDS:= \
 	simplejson chardet dateutil mysql pytz qrcode requests requests-oauthlib \
-	django django-constance
+	django django-constance django-appconf django-compressor django-formtools \
+	django-jsonfield django-picklefield django-postoffice django-restframework
 
 define Package/seafile-seahub
   SECTION:=net
   CATEGORY:=Network
   TITLE:=Seafile server - seahub component
   URL:=https://seafile.com/
-  DEPENDS:=+python +pillow +django-appconf \
-	+django-compressor +django-formtools +django-jsonfield \
-	+django-picklefield +django-postoffice +django-restframework \
+  DEPENDS:=+python +pillow \
 	+django-simple-captcha +django-statici18n +django-webpack-loader \
 	+flup +gunicorn +openpyxl \
```

Comment:
You'll need to increase `PKG_RELEASE`, since this changes the install-time dependencies, due to the package renames.  Also, you'll need to adjust to #9328 (I'm counting on it being merged soon, since it is a trivial fix).